### DISCUSSION TITLE
Failure uploading a model from civit.ai

### DISCUSSION THREAD

**Discussion by paul18890** [lllyasviel/Fooocus#2884] [Q&A]
Hello, i dont know what is the provblem but if i wanna uploading a checkpoint model, nothing happen in googla colab. i tried to upload a juggernaut checkpoitn sdxl 1.0: https://civitai.com/models/133005/juggernaut-xl . But i got a red circle in the upload field. what is the problem? 
![1](https://github.com/lllyasviel/Fooocus/assets/163112664/ad803bf3-43a6-4e4f-ad4c-2a16bd076749)

**Comment by IPv6** [comment]
The problem is civit.ai does not allow direct downloads anymore. Currently only hugginface is ok with that.

There is a workaround - you can mount colab to your google drive, then download civit.ai model manually and place it into proper folder. In that case even if colab goes down your downloaded files will stay in google drive

**Reply by paul18890** [reply]
but this is, what i have done. i downloaded the model directly from civit.ai to my desktop and i tried to upload it in colab. there came the red circle.

**Reply by IPv6** [reply]
I see, sorry for confusion. In this case it`s an internet connection problem with colab itself, huge files can be a problem // Probably you have to try again or ensure your connection is stable
